_events/sr2019/southampton-kickstart.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Southampton Kickstart
+date: 2018-11-10 10:00:00
+layout: event
+type: kickstart
+location: Southampton University
+---
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced and robotics kits will be handed out to the teams.
+
+If you are unable to attend Kickstart, please [let us know][teams-contact] so we
+can arrange another way to get you your kit.
+
+## Directions
+
+* How to [get to campus][soton-campus-directions]
+
+* [On-campus directions][soton-directions]
+
+* [Where to find food][soton-food-map]
+
+## Schedule
+
+| Time  | Info |
+|-------|------|
+| 10:00 | Introductory presentation, where the game of SR2019 will be announced. |
+| 11:00 | Team kits will be handed out, and teams will take part in activities to get them acquainted with their kits. |
+| 12:30 | There will be a lunch break. Teams are expected to bring their own lunches or get them from food vendors near the venues. |
+| 17:00 | Finish. |
+
+[teams-contact]: mailto:teams@studentrobotics.org
+[soton-food-map]: TODO
```

Comment:
For this and the below links, best copy the links from https://studentrobotics.org/events/sr2017/southampton-tech-day-february/.

Reply:
In theory yes, however they say 2017 all over them.

Reply:
I've made a new map for you here: https://drive.google.com/open?id=1vDqr-yQh-OO7MzVa_OiuKD5xUrEsLprr&usp=sharing